### ISSUE TITLE
Azure SignalR broken in preview6

### ISSUE THREAD

**Issue by davidfowl** [Azure/azure-signalr#517]
This is a placeholder issue for when we ship preview6 (which is a while away) but we made a breaking change that affects routing which breaks Azure SignalR:

![image](https://user-images.githubusercontent.com/95136/57022986-b2070c80-6be5-11e9-81aa-a90f54bac11d.png)

We just need to re-compile against the newer API (it's source compatible).

**Comment by JialinXin** [comment]
Pending update for KoreBuild updating to preview6.

**Comment by davidfowl** [comment]
It's broken differently now as we changed the signature but the fix is trivial 😄

**Comment by davidfowl** [comment]
cc @vijayrkn

**Comment by vijayrkn** [comment]
If the fix is available, Is there a private package available to test? We dont have much time left for Preview6.

**Comment by JialinXin** [comment]
@vijayrkn We're working on this. We found a few more interface changes in latest preview6 which requires Azure SignalR SDK update accordingly. We'll publish a new package when fixed all and keep you updated.

**Comment by davidfowl** [comment]
Is there a branch I can look at?

**Comment by JialinXin** [comment]
Thank you @davidfowl , please check branch [jixin/preview6](https://github.com/Azure/azure-signalr/tree/jixin/preview6) for sdk side changes.

**Comment by JialinXin** [comment]
@davidfowl Install latest .NET core sdk and the protocol issue is not exists anymore. But still we'll fix the add fallback protocol check in runtime.

**Comment by JialinXin** [comment]
@vijayrkn [`1.1.0-corepreview6-00000`](https://www.myget.org/feed/azure-signalr-dev/package/nuget/Microsoft.Azure.SignalR/1.1.0-corepreview6-00000) is published to MyGet to unblock your tests. Please let me know if you meet any problem.
Keep the issue open until it's catching up with cicd.

**Comment by vijayrkn** [comment]
@phil-allen-msft 

Can you please try this package and see if you run into any issues?

**Comment by phil-allen-msft** [comment]
I've tried out the scenarios and everything seems fine.

**Comment by davidfowl** [comment]
There may be one more late change to react to @JialinXin

**Comment by JialinXin** [comment]
@davidfowl Thanks for the update. Will follow up on this.

**Comment by JialinXin** [comment]
preview6 version release and close the issue.

**Comment by davidfowl** [comment]
We need to do the same for preview 7 let’s open a new issue to track it
